examples/views/dynamic_user_dropdown.py

```diff
@@ -0,0 +1,86 @@
+import discord
+
+bot = discord.Bot()
+
+# Defines a custom user specific dynamic dropdown
+# that the user can choose. The callback function
+# in the class is strictly to prevent the dreaded
+# interaction failed. The results are returned to
+# the calling function for further processing.
+
+
+def external_user_specific_list_function(discord_id):
+    # Replace this example with whatever your source is
+    # for the users dynamically generated items
+    users_menu_items = []
+    for i in range(5):
+        users_menu_items.append(
+            f"Item #{i + 1}) - Dynamically Generated for <@{discord_id}>"
+        )
+        i += 1
+    return users_menu_items
```

Comment:
While this works, it would be simpler to take a random sample of items out of a list (or similar)
I personally just don't like it that much

Reply:
This PR is rough, but using for i in range and then also manually increasing i is the cherry on top...

Reply:
return [f"Item #{i} - Dynamically Generated for <@{discord_id}>" for i in range(1, 6)]